src/Hosting/Hosting/src/Builder/ApplicationBuilderFactory.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+#nullable enable
```

Comment:
Since we no longer have the ref project, we can turn on nullability on individual files. Convenient!